frontend/src/scenes/ingestion/panels/PanelComponents.tsx

```diff
@@ -1,13 +1,12 @@
-import clsx from 'clsx'
 import { useActions, useValues } from 'kea'
-import { IconChevronRight } from 'lib/components/icons'
 import { LemonButton } from 'lib/components/LemonButton'
 import { LemonDivider } from 'lib/components/LemonDivider'
 import { eventUsageLogic } from 'lib/utils/eventUsageLogic'
 import React from 'react'
 import { BOOKMARKLET } from '../constants'
 import { ingestionLogic } from '../ingestionLogic'
 import './Panels.scss'
+import { ArrowLeftOutlined } from '@ant-design/icons'
```

Comment:
Boooo 😉

I will follow up at some point and replace all instances of this with a custom icon if it is used in Figma a lot

Reply:
👻

Reply:
That icon doesn't exist in the design system, not my fault 😭